doc/rucio-server.md

```diff
@@ -0,0 +1,590 @@
+Setting up a Rucio server
```

Comment:
Thanks, this is very nice! 😄 
I think it should go to https://github.com/rucio/documentation though, since all the documentation goes there.
We need to find a good point to embed it though, since this is just one way how to do it. (The container one being a different one)